src/main/java/org/folio/bulkops/service/LogFilesService.java

```diff
@@ -92,4 +70,34 @@ public void deleteFileByOperationIdAndName(UUID operationId, String fileName) {
     remoteFileSystemClient.remove(csvPath, jsonPath);
     log.info("Deleted: {}, {}", csvPath, jsonPath);
   }
-}
+
+  public void removeTriggeringAndMatchedRecordsFiles(BulkOperation bulkOperation) {
+    if (isNotEmpty(bulkOperation.getLinkToTriggeringCsvFile())) {
+      remoteFileSystemClient.remove(bulkOperation.getLinkToTriggeringCsvFile());
+      bulkOperation.setLinkToTriggeringCsvFile(null);
+    }
+    if (isNotEmpty(bulkOperation.getLinkToMatchedRecordsCsvFile())) {
+      remoteFileSystemClient.remove(bulkOperation.getLinkToMatchedRecordsCsvFile());
+      bulkOperation.setLinkToMatchedRecordsCsvFile(null);
+    }
+    if (isNotEmpty(bulkOperation.getLinkToMatchedRecordsJsonFile())) {
+      remoteFileSystemClient.remove(bulkOperation.getLinkToMatchedRecordsJsonFile());
+      bulkOperation.setLinkToMatchedRecordsJsonFile(null);
+    }
+    if (isNotEmpty(bulkOperation.getLinkToMatchedRecordsErrorsCsvFile())) {
+      remoteFileSystemClient.remove(bulkOperation.getLinkToMatchedRecordsErrorsCsvFile());
+      bulkOperation.setLinkToMatchedRecordsErrorsCsvFile(null);
+    }
+  }
+
+  public void removeModifiedFiles(BulkOperation bulkOperation) {
```

Comment:
Looks we already have such implementation https://github.com/folio-org/mod-bulk-operations/blob/f5e3eeaa91c9c1e9386cc5732c150b5af7d0460a/src/main/java/org/folio/bulkops/service/BulkOperationService.java#L505

Reply:
Please, verify and refactor if needed to avoid duplication.